tests/llmcompressor/transformers/sparsification/test_compress_tensor_utils.py

```diff
@@ -672,7 +672,13 @@ def test_correct_compressor_inferred(
     if is_24:
         weights = _make_24_sparse(weights)
     else:
+<<<<<<< HEAD
         weights[0, :] = torch.ones((4,))  # guarantee not 24 sparse
+=======
+        weights[0, :] = torch.ones(
+            4,
+        )  # guarantee not 24 sparse
+>>>>>>> 35f1d500 (black/isort)
```

Comment:
Looks like some leftover cruft from conflict resolution?

Reply:
blah, sorry, forgot to save before `git commit`